README.md

```diff
@@ -51,6 +51,46 @@ This document provides the information needed to create a DASH packager that is
 
   See https://github.com/google/edash-packager/blob/master/CONTRIBUTING.md for details.
 
+# Using docker for testing/development #
+
+> Docker allows you to package an application with all of its dependencies into a standardized unit for software development.
+>
+> -- <cite>Official site</cite>
+
+If you want try or test `edash-packager` you can just use this.
```

Comment:
How about replacing line 56 to line 60 with:

[Docker] (https://www.docker.com/whatisdocker) is a tool that can package an application and its dependencies in a virtual container that can run on different host operating systems.